src/commands/compile/spec/compile.spec.ts

```diff
@@ -453,7 +453,6 @@ describe('sasjs compile outside project', () => {
     })
 
     afterEach(async () => {
-      await deleteFolder(parentOutputFolder)
```

Comment:
Why you've decided to remain `parentOutputFolder `?

Reply:
There's no usage of `parentOutputFolder` in this `describe` block

Reply:
And deleting such non-existing folder in windows, raise exception